### ISSUE TITLE
Treesitter completion source for mini.completion

### ISSUE THREAD

**Issue by 231tr0n** [echasnovski/mini.nvim#1320]
### Contributing guidelines

- [X] I have read [CONTRIBUTING.md](https://github.com/echasnovski/mini.nvim/blob/main/CONTRIBUTING.md)
- [X] I have read [CODE_OF_CONDUCT.md](https://github.com/echasnovski/mini.nvim/blob/main/CODE_OF_CONDUCT.md)

### Module(s)

mini.completion

### Description

Hi echasnovski! Is it possible to have treesitter completion added to mini.completion something like what this plugin does for nvim-cmp [ray-x/cmp-treesitter](https://github.com/ray-x/cmp-treesitter).
This will help me use mini.completion when there is no LSP attached.
Basically my request is that if LSP is attached use LSP completion other wise use treesitter completion.

**Comment by echasnovski** [comment]
'mini.completion' has [two stage completion](https://github.com/echasnovski/mini.nvim/blob/0a8a1072137d916406507c941698a4bfa9dbbe7a/doc/mini-completion.txt#L17-L30): first try LSP and fallback to fallback source (even if LSP is attached but returned no results). You can try configuring [fallback action](https://github.com/echasnovski/mini.nvim/blob/0a8a1072137d916406507c941698a4bfa9dbbe7a/doc/mini-completion.txt#L224), but how exactly it should be done to have suggestions from tree-sitter parsing I'll leave up to you (or someone else) to figure out.
